src/Common/AppStatus/utils.ts

```diff
@@ -1,5 +1,4 @@
 import { TIMELINE_STATUS } from './constants'
-
```

Comment:
Try to have a blank linke between component and import statements for readability purpose